lib/friendly_shipping/services/ups.rb

```diff
@@ -63,10 +63,12 @@ def carriers
 
       # Get rates for a shipment
       # @param [Physical::Shipment] shipment The shipment we want to get rates for
+      # @param [FriendlyShipping::Services::Ups::RateEstimateOptions] options What options
+      #    to use for this rate estimate call
       # @return [Result<ApiResult<Array<Rate>>>] The rates returned from UPS encoded in a
       #   `FriendlyShipping::ApiResult` object.
-      def rate_estimates(shipment, debug: false)
-        rate_request_xml = SerializeRatingServiceSelectionRequest.call(shipment: shipment)
+      def rate_estimates(shipment, options:, debug: false)
```

Comment:
Any reason you don't want to default `options:` to `RateEstimateOptions.new`?

Reply:
Because `RateEstimateOptions` has a mandatory argument, the `shipper_number`. We can't supply that in a default object...